java/src/main/java/lesson01/solution/Hello.java

```diff
@@ -32,8 +32,8 @@ public static void main(String[] args) {
             throw new IllegalArgumentException("Expecting one argument");
         }
         String helloTo = args[0];
-        Tracer tracer = Tracing.init("hello-world");
-        new Hello(tracer).sayHello(helloTo);
-        tracer.close();
+        try (JaegerTracer tracer = Tracing.init("hello-world")) {
```

Comment:
IMO, the tutorial should be using OT's Tracer instead of JaegerTracer as much as possible. The side effect is that, at least for now, we'd need to close the tracer manually (see https://github.com/jaegertracing/jaeger-client-java/issues/544).

Reply:
That said, we could change the tutorial later, once we fix the mentioned issue.

Reply:
I absolutely agree! I have changed the code and lesson01/README.md to reflect this.